renpy/display/layout.py

```diff
@@ -573,6 +567,21 @@ def per_interact(self):
         for _ in range(delta):
             self.add(null)
 
+    def generate_gridmap(self):
+        row_order = list(range(self.rows))
```

Comment:
No need to convert the range to a list here.
```suggestion
        row_order = range(self.rows)
```